ci/jenkins/jobs/projects.yaml

```diff
@@ -377,3 +377,162 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{test_name}-daily':
+          test_name: nephe-test-aws
+          node: 'nephe'
+          description: 'This is the daily {test_name}.'
+          branches:
+            - '${{NEPHE_GIT_REVISION}}'
```

Comment:
I think it should be {NEPHE_GIT_REVISION}

Reply:
I'm using it from one of the jenkins job in antrea.

Reply:
let's keep it as it is now, I made a mistake

Reply:
Sure